<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>334</width>
    <height>467</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QRadioButton" name="rbTcpSocket">
         <property name="text">
          <string>TCP socket</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rbSslSocket">
         <property name="text">
          <string>SSL socket</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="pbConnect">
           <property name="text">
            <string>Connect</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbDisconnect">
           <property name="text">
            <string>Disconnect</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbClear">
           <property name="text">
            <string>Clear</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLineEdit" name="lePort">
           <property name="text">
            <string>443</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="leHost">
           <property name="text">
            <string>www.naver.com</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="pteMessage">
         <property name="plainText">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QPlainTextEdit" name="pteMesage2">
           <property name="plainText">
            <string>GET / HTTP/1.1
Host:www.naver.com

</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbSend">
           <property name="text">
            <string>Send</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
